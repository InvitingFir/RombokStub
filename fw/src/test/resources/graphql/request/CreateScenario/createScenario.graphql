mutation createScenario($params: ScenarioRequest!, $uuid: String!){
  createScenario(params: $params, serviceUuid: $uuid){
    scenarioType
    name
    uuid
    predicate
    action
    isDefault
    variables {
      name
      value
    }
    url
  }
}