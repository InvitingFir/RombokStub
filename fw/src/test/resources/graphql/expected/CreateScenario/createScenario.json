{
  "data": {
    "createScenario": {
      "scenarioType": "HTTP",
      "uuid": "9161f68d-da9e-4fa2-b0ad-142bce3b4550",
      "action": "action",
      "isDefault": false,
      "name": "name",
      "predicate": "predicate",
      "url": "url"
    }
  }
}